analysis/import/SonarImporter/src/main/kotlin/de/maibornwolff/codecharta/importer/sonar/SonarImporterMain.kt

```diff
@@ -93,4 +95,26 @@ class SonarImporterMain(
     }
 
     override fun getDialog(): ParserDialogInterface = ParserDialog
+    override fun isUsable(inputFile: String): Boolean {
+        val trimmedInput = inputFile.trim()
+        // TODO: Is this sufficient to check for urls?
+        if (trimmedInput.startsWith("http://") || trimmedInput.startsWith("https://")) {
+            return true
+        }
+
+        if (trimmedInput.endsWith("sonar-project.properties")) {
```

Comment:
Just curious... Why using `endsWith?`

Reply:
Well if a project contains this file it is very probable that it is a sonar analyzed project, so suggesting sonar made sense to me.

About your second comment: I just copy pasted it from the GitLogParser and forgot to change the method